### ISSUE TITLE
Cleanup file structure for deleted files

### ISSUE THREAD

**Issue by alliomeria** [esmero/webform_strawberryfield#83]
# What is this? :bug:

When files are deleted from an ADO, and a certain key becomes empty (such as "image"), the corresponding file structures ("as:image") are sticking around even after the file deletion is saved. 

This issue may have been introduced near Beta3; Related to `strawberryfield` module.

**Comment by DiegoPino** [comment]
True. This is on me and I introduced this when I added (for drush ingest) the as:filetype structure generator as an event subscriber but stripped the old code from that needed pieces. There is even a comment from me to me.. gosh. Will fish asap. Will do it right because we do not play with people's files. Thanks a lot @alliomeria !
